app/views/mentor_overview/index.html.haml

```diff
@@ -8,7 +8,7 @@
     = current_user.first_name
     = current_user.last_name
 
-- if Time.zone.now < Time.new(2018, 9, 1).in_time_zone && current_user.protocol_subscriptions.active.count.positive?
+- if Time.zone.now < Time.new(2018, 10, 1).in_time_zone && current_user.protocol_subscriptions.active.count.positive?
```

Comment:
Misschien kunnen we hier iets definitievers van maken? Bijv. de time check weghalen? Of De time check > dan een bepaalde datum laten zijn ipv kleiner dan een bepaalde datum? Anders moeten we namelijk iedere maand deze regel aanpassen.